package.json

```diff
@@ -1,6 +1,8 @@
 {
   "scripts": {
-    "test-daemon": "if [ -d ./daemon-build ]; then mocha --opts tests/mocha.opts tests/**/*test.js; else echo \"Please run yarn link-daemon first.\"; exit 1; fi",
+    "test": "if [ -d ./daemon-build ]; then mocha --opts tests/mocha.opts tests/**/*test.js; else echo \"Please run yarn link-daemon first.\"; exit 1; fi",
+    "test-functional": "if [ -d ./daemon-build ]; then mocha --opts tests/mocha.opts tests/functional/*test.js; else echo \"Please run yarn link-daemon first.\"; exit 1; fi",
+    "test-integration": "if [ -d ./daemon-build ]; then mocha --opts tests/mocha.opts tests/integration/*test.js; else echo \"Please run yarn link-daemon first.\"; exit 1; fi",
```

Comment:
Did you intend to switch everything over to `npm`? Some stuff still uses `yarn`.

Reply:
Nice catch! Fixed.